prime-router/docs/onboarding-users/senders.md

```diff
@@ -5,22 +5,172 @@ Note for content writer: If there is overlap with the previous "Pipeline Configu
 
 ## Sender Configuration
 
-- Validation Tool
- - Via ReportStream website
-   - Via API
-   
+### Get Ready
+
+Create a new branch to store your work on the new sender.
+
+### Set up a New Organization
+
+In your `organization.yml` file create a new organization.
+
+Example organization with one sender entry:
+
+```yaml
+- name: yoyodyne
+  description: Yoyodyne Propulsion Laboratories, the Future Starts Tomorrow!
+  jurisdiction: FEDERAL
+  senders:
+    - name: default
+      organizationName: yoyodyne
+      topic: full-elr
+      schemaName: metadata/fhir_transforms/senders/default-sender-transform
+      format: FHIR
+```
+
+A few things to note here:
+
+- The name of the organization must be unique. It cannot share a name with a state or other organization
+- The jurisdiction should be FEDERAL since they are not a STLT
+- The organizationName under `senders` must match the name of the org above
+- The format here is `FHIR`, though it is possible it could another format, such as `HL7`
+- The schema name should use `metadata/fhir_transforms/senders/default-sender-transform` by default
+- Pay attention to the value in the topic field to make sure it is correct for your use case
+
+### Set up a New Schema
+
+**You should only create new schemas when an existing schema will not fit your use case. Please ensure that an
+existing schema cannot be used prior to creating a new schema.**
+
+Once you've added the sender to the `organizations.yml` file you next need to create a schema file.
+
+The schema provides additional transforms required so that the senders data can be routed correctly.
+
+Note: When creating a transform, please reference [Changing/Updating Sender/Receiver Transforms](../getting-started/standard-operating-procedures/changing-transforms.md) for guidance.
+
+## Testing
+
+### Note
+In order to test a sender, you must create (or have in your possession) a file that matches the format
+you have laid out in the schema, and that matches the format the sender will provide.  Also, it is
+good to have at least one receiver that will match what the sender provides. For example, if Yoyodyne
+is sending information for a lab in NJ, but you have no receivers set up to receive information
+from NJ, then you will be unable to complete testing. The sender and receiver also need to have the same topic.
+
+### Testing Locally
+Assuming you have a receiver set up for your sender, follow the below steps to build and test your solution:
+
+```
+./git checkout “branch_name”
+./gradlew clean package
+```
+
+
+Once that has completed successfully, the next step is to manually check the Universal Pipeline can process 
+the message with no errors and not lose any data while converting it.
+
+#### Testing sender transforms
+```shell
+./prime fhirdata --input-file "PATH-TO-SAMPLE-FILE" -s metadata/fhir_transforms/senders/default-sender-transform.yaml --output-format FHIR --output-file "PATH-TO-OUTPUT-FILE"
```

Comment:
This command does test our HL7 to FHIR transforms. However, right now it looks like we don't have the ability to pass a sender transform schema when going from HL7->FHIR , but it does work when doing HL7->FHIR->HL7. I looked over the tickets and this should've been added as part of this ticket [10191](https://app.zenhub.com/workspaces/engagement-6166edbd409257001e09f1b2/issues/gh/cdcgov/prime-reportstream/10191). I'll create a ticket for it and get it fixed.

Reply:
But since this command specifies the output as FHIR and it doesn't look at the schema when going from HL7 -> FHIR, it wouldn't test the transform. I think I have an easy fix for that... (Do HL7 -> FHIR -> FHIR if we want it to look at the schema specified with -s)